### ISSUE TITLE
I am at a loss here.

### ISSUE THREAD

**Issue by Mellonoma420** [FrSkyRC/ETHOS-Feedback-Community#2210]
I have a Jumper AION Nano 2400 TX ELRS module. Version 3.0 of ELRS. I put the lua script on the SD card for my X18.

I was using it my Taranis X9 lite on the MATEK R-2400-D with no issues all month, and just added a new receiver to it, so I know it's all working correctly.

I have a Tandem X18, firmware version 1.4.3 of Ethos. Just got it in the mail this morning.

Ethos is an entirely new OS for me, and on top of that I am reasonably new to this hobby. My main issue is that I don't know how to actually RUN a lua script on this radio. I've searched every screen in the thing, and can find nothing. I can find the files, but only have the option to move, copy, or delete. I have to be missing something, andd I don't know what. Someone please help, even if it is just to tell me I'm doing it wrong, or that it just doesn't work yet lol.

**Comment by mawzthefinn** [comment]
All the info you need is here:

https://www.rcgroups.com/forums/showpost.php?p=49550649&postcount=18844

In short, if you install the ELRS script in /scripts/elrs and restart the radio, then the ELRS protocol becomes available to be selected in RF System for External Modules. Once selected, you can access the module and receiver configuration pages from the Configure button in the External Module section (note receiver will only show up if on and linked)

**Comment by Mellonoma420** [comment]
Ahh, gotcha, and thanks for the response.

I do get the ELRS option, but only if I select the "Jumper T pro." I get that to work just fine, but my ELRS module displays "NO HANDSET" on it's screen, and when I press the configure button I get a message saying "Module did not respond" (or something like that.) If it just doesn't have support yet it's fine, I've decided to just use the TD MX receiver it came with. Problem with THAT is that I can't get bars to move in ardupilot lol. I can figure that one out though. Maybe.

Thanks again.

**Comment by mawzthefinn** [comment]
Are you running ELRS 3.0 on the module?

I have the HappyModel ES24TX module myself and it works well on ETHOS 1.4.1 and later, with ELRS 3.0.0 loaded on the module.

**Comment by Mellonoma420** [comment]
Yeah, it's running v3. I get a few of the FrSky external modules that I
recognize, and the Jumper T pro as the only other. If I select it I get the
ELRS option in the box next to it, but my module just says "NO HANDSET" on
the screen. Trust me, it has been frustrating as hell, because as far as I
know I've followed all the steps.

I just decided to use the TD MX receiver that came with my X18, and now
we're good to go. I've wasted a whole day. My ass hurts from my chair. I'm
printing out a back piece for the ELRS module so it will fit in the Taranis
and I just have 2 radios now. I need sleep.

Thanks for the help, I appreciate your time. I will work on this when I
wake up. Probably figure it out immediately too lol.
